### ISSUE TITLE
Issue with 67 Image Pull

### ISSUE THREAD

**Issue by praval1983** [VFrontDe/ESXi-Customizer-PS#25]
The following commands can successfully create ISO for ESXi65 build.

`
PS C:\67> [Net.ServicePointManager]::SecurityProtocol = [Net.ServicePointManager]::SecurityProtocol -bor [Net.SecurityProtocolType]::Ssl3

PS C:\67> .\ESXi-Customizer-PS.ps1 -v65 -vft -nsc -load net-igb -remove vmkusb -log C:\67\ISO\ESXi-Customizer-PS.log -outDir C:\67\ISO
`

However, the same way I am not able to create ISO for ESXi67 build. 

`
PS C:\67> [Net.ServicePointManager]::SecurityProtocol = [Net.ServicePointManager]::SecurityProtocol -bor [Net.SecurityProtocolType]::Ssl3

PS C:\67> .\ESXi-Customizer-PS.ps1 -v67 -vft -nsc -load net-igb -remove vmkusb -log C:\67\ISO\ESXi-Customizer-PS.log -outDir C:\67\ISO
`

See successful log for 65 build and un-successful 67 builds in the attached log.zip file.

[Log.zip](https://github.com/VFrontDe/ESXi-Customizer-PS/files/4944734/Log.zip)

**Comment by virtualex-itv** [comment]
Did you reboot after running the command to enable Ssl3?

**Comment by praval1983** [comment]
Yes. I did reboot my computer after enabling ssl3 it doesn't change the outcome.
just to clear any doubt. I executed above command to enable ssl3 and rebooted and tried again and still I have same error while creating 6.7 iso.

It's surprising to me that the same computer same configuration and I can create 6.5 ISO without any issue.

**Comment by virtualex-itv** [comment]
i believe this has something to do with the driver you are trying to load into the image not being supported on any version of vSphere higher than `-v65`.  Most, if not all of the drivers available via `-vft -load` are legacy and have not been maintained or lack support for newer versions of vSphere due to architectural changes.
